src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -207,7 +209,20 @@ public static void RoundtripString()
             Assert.Equal("12-34-56-78-9A", BitConverter.ToString(bytes));
             Assert.Equal("56-78-9A", BitConverter.ToString(bytes, 2));
             Assert.Equal("56", BitConverter.ToString(bytes, 2, 1));
-            Assert.Equal(String.Empty, BitConverter.ToString(new byte[0]));
+
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[0]));
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[3], 1, 0));
```

Comment:
When viewed as "are we doing what we want us to do?" it seems right.  When viewed as "should a user feel comfortable depending on this?" it feels a bit strong.  And since it's immutable (ish) it isn't codifying Dispose warnings or anything like that...

So, I'm not saying it's wrong.. just that I'd've written Equal and I'm indirectly wondering what I'm missing.